presto-main/src/main/java/io/prestosql/testing/TestingSession.java

```diff
@@ -89,6 +89,13 @@ private static Connector createTestSessionConnector()
     {
         return new Connector()
         {
+            @Override
+            public boolean isSingleStatementWritesOnly()
```

Comment:
I don't think we need this implementation, since these connector methods are never called, and thus the default implementation works.